lib/constants.js

```diff
@@ -7,6 +7,7 @@ REOL_CHAN_ID  = '296232884623376384'
 ANNOUNCEMENTS_ID = '413504832356352020'
 PHOTO_GALLERY_ID = '423787873448493056'
 DISCOGRAPHY_ID = '515055814755614730'
+COLORS = ["Red", "Blue", "Green", "Purple", "Yellow", "Orange", "Pink"]
```

Comment:
The roles we were planning for are actually region based:
```["Scandinavian", "North America", "EU - UK", "EU - Germany", "Asia - SEA", "Asia - Japan", "Australia", "South America"]```
Could you change the roles to these?
I'm not sure how would spaces affect the commands, so you should try those out too

Reply:
Sorry, we can leave the regions to another PR